Automattic/wp-calypso
# Hosting Command Palette: Refactor the command to open phpMyAdmin in Calypso

Related to https://github.com/Automattic/dotcom-forge/issues/4612

## Proposed Changes

This PR refactors the command for opening phpMyAdmin. Instead of adding the code for the code to do API request again, it reuses `openPmaLink` function from `calypso/my-sites/hosting/phpmyadmin-card`.

## Testing Instructions

* Navigate to Calypso Live link
* Navigate to `/sites`
* Press `cmd + k` on macOS or `ctrl + k` on Windows to trigger command palette
* Select the command to `Open database in phpMyAdmin`
* Select a site for which you want to open the database
* Confirm that phpMyAdmin opens
* Navigate to `/hosting-config/{yourAtomicSite}`
* Open phpMyAdmin from the Hosting Configuration and confirm it works as expected

**Notes**

* I tried both approaches mentioned https://github.com/Automattic/wp-calypso/pull/84376#discussion_r1403169584 and using a hook worked out better than splitting a function as it was not breaking the rule of hooks (the issues that I was running into with a splitting a function were related to `useDispatch` and `useTranslate` and when they were being used)

## Pre-merge Checklist
